src/routes/safe/container/index.tsx

```diff
@@ -49,10 +45,6 @@ const Container = (): React.ReactElement => {
   const matchSafeWithAddress = useRouteMatch<{ safeAddress: string }>({ path: `${SAFELIST_ADDRESS}/:safeAddress` })
   const safeAppsEnabled = Boolean(featuresEnabled?.includes(FEATURES.SAFE_APPS))
 
-  if (!safeAddress) {
```

Comment:
This code is obsolete, there is no way to reach this page without a safeAddress.